package.yaml

```diff
@@ -57,6 +57,7 @@ library:
   dependencies:
   - natural-transformation >= 0.2
   - transformers-base
+  - template-haskell
```

Comment:
This dependency should have version bounds, since TH is notoriously backwards-incompatible.